recipes/fluidimage/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set name = "fluidimage" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 749d682ea0a14c9d9e7d58c8b0d4a0274a89c3fa2a0a81557563473b85236d4d
+
+build:
+  number: 0
+  entry_points:
+    - fluidimviewer-pg = fluidimage.gui.pg_main:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - fluiddyn >=0.2.0
+    - h5netcdf
+    - h5py
```

Comment:
Are these dependencies needed at build time. Odds are you just need python pip and setuptools_scm

Reply:
I deleted all the unnecessary dependencies